lib/scrabble.rb

```diff
@@ -1,4 +1,8 @@
 module Scrabble
   # require your gems and classes here
-  # require_relative 'scrabble/whatevs'
+  require 'pathname'
+
+  ROOT_DIR = Pathname(__dir__).parent ## QUESTION: Is Pathname(__dir__) == Pathname.new(__dir__) ?
```

Comment:
When I tried it, it worked the same way. I just wasn't sure if there was a preferred way, or if there are certain situations where it wouldn't work the same.